blog/posts/views.py

```diff
@@ -0,0 +1,110 @@
+import json
+from django.http import JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+from .models import PostModel 
+from django.views.generic import View
+from django.utils.decorators import method_decorator
+from users.models import UserModel
+from users.views import isAuthenticated
+
+@method_decorator(csrf_exempt, name='dispatch')
+class PostHandler(View):
+    def post(self, request):
+        response = {}
+        post_data = json.loads(request.body.decode('utf-8'))
+
+        # validate post_data
+        if post_data is None or len(post_data) != 3:
+            return JsonResponse({'message' : 'Invalid post'})
```

Comment:
validation should also include the length of the message of the message within post data.